test/utils/helpers.go

```diff
@@ -376,6 +401,51 @@ func GenerateNameFromText(fullText string, focus []string) string {
 	return strings.ReplaceAll(fullText, " ", "")
 }
 
+// SetAgentConfig configures the cluster, generates agent kubeconfigs and sets terraform env vars.
+func SetAgentConfig(c client.Client, ns *corev1.Namespace, cloudProviders, antreaVersion, kubeconfig string) error {
+	err := c.Create(context.TODO(), ns)
+	if err != nil {
+		return fmt.Errorf("failed to create static vm namespace %+v", err)
+	}
+
+	// TODO: decouple cluster type with cloud provider type
+	clusterType := ""
+	switch cloudProviders {
+	case string(cloudv1alpha1.AzureCloudProvider):
+		clusterType = "aks"
+	case string(cloudv1alpha1.AWSCloudProvider):
+		clusterType = "eks"
+	}
+
+	dir, err := os.UserHomeDir()
+	if err != nil {
+		return err
+	}
+	dir = dir + "/tmp/integration/"
+
+	err = os.MkdirAll(dir, 0777)
+	if err != nil {
+		return err
+	}
+	_ = os.Setenv("KUBECONFIG", kubeconfig)
+	cmd := exec.Command("./ci/generate-agent-config.sh", "--cluster-type", clusterType, "--antrea-version", antreaVersion,
+		"--target-dir", dir)
```

Comment:
can we also pass namespace here?

So that we could dyamically update the namespace in vm rbac file, in future.